caluma/form/jexl.py

```diff
@@ -15,8 +15,21 @@ class QuestionJexl(JEXL):
     def __init__(self, answer_by_question={}, **kwargs):
         super().__init__(**kwargs)
 
-        self.add_transform("answer", lambda question: answer_by_question.get(question))
+        self.context = answer_by_question
+        self.add_transform("answer", self.answer_transform)
         self.add_transform("mapby", lambda arr, key: [obj[key] for obj in arr])
 
+    def answer_transform(self, *args):
```

Comment:
I'd prefer explicit arguments - they have specific meanings anyway, judging from the code.

You can do `def answer_transform(self, question, path=None):` to get the same effect

Reply:
Right, that's what I was looking for! My python is a bit rusty :blush:

Reply:
Fixed